BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java

```diff
@@ -88,8 +88,8 @@ private void updateLocation() {
         Multilateration multilateration = new Multilateration(usableBeacons);
         Location location = multilateration.getLocation();
 
-        // use deviation of multilateration to stabilize location
-        if (multilateration.getDeviation() < 1) {
+        // use root mean square of multilateration to stabilize location
+        if (multilateration.getRMS() < 10) {
```

Comment:
We should maybe add constants to define understandable thresholds like `strict`, `medium` etc. (instead of magic number).